internal/contour/holdoff.go

```diff
@@ -76,9 +76,6 @@ func (hn *HoldoffNotifier) OnChange(builder *dag.Builder) {
 // counter holds an atomically incrementing counter.
 type counter uint64
 
-func (c *counter) get() uint64 {
```

Comment:
Why did this get deleted?

Reply:
The static check tool was complaining that this function was not being used. And since inc() and reset() themselves return the updated value, get() was not being used in the code.